access-control-app/resources/templates/access-control-base.html

```diff
@@ -2,4 +2,11 @@
 
 {% block app-nav %}
   <li><a href="{{ base-url }}site/docs/access-control">Documentation</a></li>
+  <script src="https://cdn.uat.earthdata.nasa.gov/tophat2/tophat2.js"
```

Comment:
Now that I better understand what Tophat2 is, I think there's a better way to do this -- you're close, we just need to generalize a bit so there's not so much duplication in the code.

Let's do this:
* Update `template/base.html` with a new, empty block `{% block tophat-js %}{% endblock %}`
* Create a new template file called `base-tophat.html` (in the `resources/templates` directory) which extends `templates/base.html` and put the script tophat setup there in the new block
* Remove the script from all the other templates
* Update the `*-home*` templates to extend `base-tophat.html` instead of `base.html`

This will mean there only being one declaration of the script setup.

If the template stuff is out of your comfort zone, you can add me as a collaborator on your branch, and I can make commits to it ...